sentry_sdk/client.py

```diff
@@ -151,13 +151,88 @@ def _get_options(*args, **kwargs):
     module_not_found_error = ImportError  # type: ignore
 
 
-class _Client(object):
+class NoopClient:
+    """
+    A client that does not send any events to Sentry. This is used as a fallback when the Sentry SDK is not yet initialized.
+
+    .. versionadded:: 1.XX.0
+    """
+
+    options = _get_options()  # type: Dict[str, Any]
+    metrics_aggregator = None  # type: Optional[Any]
+    monitor = None  # type: Optional[Any]
+    transport = None  # type: Optional[Any]
```

Comment:
Can we use a more specific type than `Optional[Any]`? For example, the transport could probably be marked as `Optional[Transport]`.

Reply:
done.